dts/arm/nxp/nxp_rw6xx_common.dtsi

```diff
@@ -300,8 +296,7 @@
 		resets = <&rstctl1 NXP_SYSCON_RESET(0, 22)>;
 		dmas = <&dma0 26>, <&dma0 27>;
 		dma-names = "rx", "tx";
-		zephyr,disabling-power-states = <&suspend &standby>;
-		power-domains = <&power_mode3_domain>;
+		power-domains = <&power_mode2_domain>;
```

Comment:
looks like there are a few holdovers with a nonexistent domain

Reply:
ah, just did a replace all for `power_mode3_domain`, forgot the 2

Reply:
fixed